packages/embed/src/utils/create-http-warning-banner.ts

```diff
@@ -0,0 +1,34 @@
+export const createHttpWarningBanner = () => {
+  const url = new URL(window.location?.href)
+  const isWarningNeeded = url.protocol !== 'https' && url.hostname !== 'localhost'
+
+  if (isWarningNeeded) {
+    showConsoleWarning()
+  }
+
+  const result = isWarningNeeded ? createBannerElement() : createFakeBannerElement()
+
+  return result
+}
+
+const createBannerElement = () => {
+  const el = document.createElement('div')
+  el.className = 'http-warning-banner'
+  el.innerText = getWarningMessage()
+
+  return el
+}
+
+const createFakeBannerElement = () => document.createElement('div')
+
+const showConsoleWarning = () => {
+  const url = new URL(window.location?.href)
+  if (url.protocol !== 'https' && url.hostname !== 'localhost') {
+    // eslint-disable-next-line no-console
+    console.error(getWarningMessage())
+  }
+}
+
+const getWarningMessage = () =>
+  `ERROR: Typeform forms can be embedded only on sites served over HTTPS,
```

Comment:
Unnecessary line break?

Reply:
Just so it takes less horizontal space. Do you want me to put it into the same line? It might take more horizontal space, but should be okay overall too.

Reply:
I would keep it as single line. If you are bothered by a long line in code maybe you can use standard quotes and split it into multiple lines with `+`. The odd line-break in error message and console is kind of bothering me :D